### PR TITLE
add the enableUserWorkloadMonitoring configmap

### PR DESCRIPTION
**Pull request by sallyom**
@JasonPowr @omveda 
I suppose we really need a test for this but it's OpenShift specific so will be a follow-up

### CODE REVIEW

File: charts/trusted-artifact-signer/ci/ci-values.yaml [hunk 1]
```diff
@@ -36,10 +38,6 @@ scaffold:
             ClientID: sigstore
             Type: email
   rekor:
-    clusterMonitoring:
-      enabled: false
-      http:
-        enabled: false
     server:
```
Comment (JasonPowr) [inline charts/trusted-artifact-signer/ci/ci-values.yaml:41]:
```
    clusterMonitoring:
      enabled: false
      http:
        enabled: false
```

I noticed that the http values are removed here but not in examples/values-kind-sigstore.yaml, is this right ?

Comment (sallyom) [reply charts/trusted-artifact-signer/ci/ci-values.yaml:41]: I think those were placed in error from the beginning : ) so I'll remove that, also, thanks

**Comment by sallyom** [comment]
> I need to modify this PR- since sometimes the monitoring cm already exists & since it's not managed by helm, helm complains

updated this to add instructions in enable-grafana-monitoring.md

File: charts/trusted-artifact-signer/ci/ci-values.yaml (hunk 1 above)
Comment (sallyom) [reply charts/trusted-artifact-signer/ci/ci-values.yaml:41]: removed!

**Pull request merged**